lib/msal-browser/src/crypto/BrowserCrypto.ts

```diff
@@ -80,11 +86,54 @@ export function getRandomValues(dataBuffer: Uint8Array): Uint8Array {
 }
 
 /**
- * Creates a new random GUID
- * @returns
+ * Creates a UUID v7 from the current timestamp.
+ * Implementation relies on the system clock to guarantee increasing order of generated identifiers.
+ * @returns {number}
  */
 export function createNewGuid(): string {
-    return window.crypto.randomUUID();
+    function getRandomUint32(): number {
```

Comment:
For consistency, anonymous function or define outside.

```suggestion
    const getRandomUint32 = (): number => {
```

Reply:
Fixed.